src/main/java/com/minecolonies/coremod/client/gui/WindowBuildTool.java

```diff
@@ -232,6 +257,40 @@
      */
     private       DialogDoneCancel confirmDeleteDialog;
 
+    /**
+     * Check if the tool is in the static schematic mode.
+     */
+    private boolean staticSchematicMode = false;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "staticSchematicMode" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1068)